### PR TITLE
Add custom values for Scalar DB helm chart

### PR DESCRIPTION
**Pull request by thongdk8**
This PR contains custom values for Scalar DB helm chart. Please take a look.

### CODE REVIEW

**Comment by tei-k** [comment]
Shouldn't we also include the ansible role and documentation for deploying `scalardb` ? (in a separate PR?)

File: conf/scalardb-custom-values.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+scalardb:
```
Comment (feeblefakie) [inline conf/scalardb-custom-values.yaml:1]: Please make it consistent with scalardl-custom-values.

File: conf/README.md [hunk 2]
```diff
@@ -4,5 +4,6 @@ Default configuration for operation directory
 
 ## Helm configuration
 
-- scalardl-custom-values.yaml
-- schema-loading-custom-values.yaml
+- [scalardl-custom-values.yaml](scalardl-custom-values.yaml)
```
Comment (feeblefakie) [inline conf/README.md:7]: Let's keep the unrelated changes unmodified.

**Review by feeblefakie — COMMENTED** [review]
Thank you!
I think it should follow what we have done for scalardl-custom-values.

**Comment by thongdk8** [comment]
Close this PR since we had new custom values for scalardb.
